### ISSUE TITLE
ter-newline-after-var does not work for nested vars

### ISSUE THREAD

**Issue by Xenya0815** [buzinas/tslint-eslint-rules#355]
### Used versions
TSC: 3.0.3
tslint: 5.11.0
tslint-reslint-rules: 5.4.0

### Used tslint.json
```json
{
  "extends": [
    "tslint-eslint-rules"
  ],
  "rules": {
    "ter-newline-after-var": [true, "always"]
  }
}
```

### Linted code
```typescript
const myFunc = () => {
  const x = 2;
  const y = 3;
  return x + y;
};
```
### Actual behavior
No linter error

Note: Same behavior when I change the arrow function to a real function: `const myFunc = function() {...}`

### Expected behavior
The linter error "Expected blank line after variable declarations."

### Analysis
If I change the `const myFunc` to a real function, the linter sees the problem:

```typescript
function myFunc() {
  const x = 2;
  const y = 3; // ERROR
  return x + y;
};
```

I'm not experienced with tslinter implementations so I don't know if this information helps:
I found out that the function `onNode` (file: dist/rules/terNewlineAfterVarRule.js) is called only once with `node.kind === ts.SyntaxKind.VariableStatement => true` for the outer `const`.

Thanks for the help!

**Comment by Xenya0815** [comment]
@webschik I checked the changes at your repository. It works, thx.
But it breaks the possibility to add additional blank lines:

```typescript
const foo = 1;  // ERROR: Unexpected blank line after variable declarations.

const bar = 2;
```

**Comment by webschik** [comment]
Thanks, @Xenya0815 !
I specifically made this case to trigger the error, but Eslint [allows it](https://eslint.org/demo/#eyJ0ZXh0IjoiY29uc3QgZm9vID0gMTtcblxuY29uc3QgYmFyID0gMjtcblxuYWxlcnQoZm9vICsgYmFyKTsiLCJvcHRpb25zIjp7InBhcnNlck9wdGlvbnMiOnsiZWNtYVZlcnNpb24iOjksInNvdXJjZVR5cGUiOiJzY3JpcHQiLCJlY21hRmVhdHVyZXMiOnt9fSwicnVsZXMiOnsiY29uc3RydWN0b3Itc3VwZXIiOjIsImZvci1kaXJlY3Rpb24iOjIsImdldHRlci1yZXR1cm4iOjIsIm5vLWNhc2UtZGVjbGFyYXRpb25zIjoyLCJuby1jbGFzcy1hc3NpZ24iOjIsIm5vLWNvbXBhcmUtbmVnLXplcm8iOjIsIm5vLWNvbmQtYXNzaWduIjoyLCJuby1jb25zb2xlIjoyLCJuby1jb25zdC1hc3NpZ24iOjIsIm5vLWNvbnN0YW50LWNvbmRpdGlvbiI6Miwibm8tY29udHJvbC1yZWdleCI6Miwibm8tZGVidWdnZXIiOjIsIm5vLWRlbGV0ZS12YXIiOjIsIm5vLWR1cGUtYXJncyI6Miwibm8tZHVwZS1jbGFzcy1tZW1iZXJzIjoyLCJuby1kdXBlLWtleXMiOjIsIm5vLWR1cGxpY2F0ZS1jYXNlIjoyLCJuby1lbXB0eS1jaGFyYWN0ZXItY2xhc3MiOjIsIm5vLWVtcHR5LXBhdHRlcm4iOjIsIm5vLWVtcHR5IjoyLCJuby1leC1hc3NpZ24iOjIsIm5vLWV4dHJhLWJvb2xlYW4tY2FzdCI6Miwibm8tZXh0cmEtc2VtaSI6Miwibm8tZmFsbHRocm91Z2giOjIsIm5vLWZ1bmMtYXNzaWduIjoyLCJuby1nbG9iYWwtYXNzaWduIjoyLCJuby1pbm5lci1kZWNsYXJhdGlvbnMiOjIsIm5vLWludmFsaWQtcmVnZXhwIjoyLCJuby1pcnJlZ3VsYXItd2hpdGVzcGFjZSI6Miwibm8tbWl4ZWQtc3BhY2VzLWFuZC10YWJzIjoyLCJuby1uZXctc3ltYm9sIjoyLCJuby1vYmotY2FsbHMiOjIsIm5vLW9jdGFsIjoyLCJuby1yZWRlY2xhcmUiOjIsIm5vLXJlZ2V4LXNwYWNlcyI6Miwibm8tc2VsZi1hc3NpZ24iOjIsIm5vLXNwYXJzZS1hcnJheXMiOjIsIm5vLXRoaXMtYmVmb3JlLXN1cGVyIjoyLCJuby11bmRlZiI6Miwibm8tdW5leHBlY3RlZC1tdWx0aWxpbmUiOjIsIm5vLXVucmVhY2hhYmxlIjoyLCJuby11bnNhZmUtZmluYWxseSI6Miwibm8tdW5zYWZlLW5lZ2F0aW9uIjoyLCJuby11bnVzZWQtbGFiZWxzIjoyLCJuby11bnVzZWQtdmFycyI6Miwibm8tdXNlbGVzcy1lc2NhcGUiOjIsInJlcXVpcmUteWllbGQiOjIsInVzZS1pc25hbiI6MiwidmFsaWQtdHlwZW9mIjoyLCJuZXdsaW5lLWFmdGVyLXZhciI6Mn0sImVudiI6eyJicm93c2VyIjp0cnVlLCJub2RlIjpmYWxzZX19fQ==). I will fix that.

**Comment by webschik** [comment]
I've updated [PR](https://github.com/buzinas/tslint-eslint-rules/pull/368)
